Add TEEP Agent and remove use of SD

mingpeiwk opened the pull request:
TEEP Agent is added into architecture diagram inside a TEE. Word Agent that was used along with TEEP Broker is updated to be Broker.

Use of SD is removed in the doc.

Commit:
Add TEEP Agent and remove use of SD

dthaler reviewed (approved):
Great progress, these changes look good

Commit:
Add Agent to diagram and defer author change

Add Agent to multiple TEE support diagram, and add links to connect broker with agent. Author change is deferred for now to post IETF meeting in two weeks, and ready to make the change.
